sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubAsyncClientIntegrationTest.java

```diff
@@ -151,17 +151,18 @@ void getPropertiesWithCredentials() {
     @Test
     public void sendAndReceiveEventByAzureNameKeyCredential() {
         ConnectionStringProperties properties = getConnectionStringProperties();
-        String fullyQualifiedNamespace = getFullyQualifiedDomainName();
+        String fullyQualifiedNamespace = properties.getEndpoint().getHost();
         String sharedAccessKeyName = properties.getSharedAccessKeyName();
         String sharedAccessKey = properties.getSharedAccessKey();
-        String eventHubName = getEventHubName();
+        String eventHubName = properties.getEntityPath();
 
         final EventData testData = new EventData(TEST_CONTENTS.getBytes(UTF_8));
 
         EventHubProducerAsyncClient asyncProducerClient = new EventHubClientBuilder()
-                .credential(fullyQualifiedNamespace, eventHubName,
-                        new AzureNamedKeyCredential(sharedAccessKeyName, sharedAccessKey))
-                .buildAsyncProducerClient();
+            .credential(fullyQualifiedNamespace, eventHubName,
+                new EventHubSharedKeyCredential(sharedAccessKeyName,
```

Comment:
Is there a reason we're updating this test that should test AzureNamedKeyCredential to EventHubSharedKeyCredential?

Reply:
The code has been rollback to the original settings and successfully passed the test on UsGov and China cloud by pipeline.